web/migrations/manifest.json

```diff
@@ -0,0 +1,32 @@
+{
+  "app": "glossy-party-1695",
```

Comment:
Note the need to rebuild / config update when changing the app id.

Reply:
sorry this slipped through... but that app doesn't exist and points out an issue:

`electric config update --app NON_EXISTING_APP` will result in modifying the `manifest.json`. Opening a ticket so we don't forget (VAX-517).